snapfaas/bins/singlevm/main.rs

```diff
@@ -335,7 +356,20 @@ fn main() {
     eprintln!("***********************************************");
 
     // Shutdown the vm and exit
-    eprintln!("Shutting down vm..."); 
+    eprintln!("Shutting down vm...");
     drop(vm);
     unlink_unix_sockets();
 }
+
+struct FSWrapper(Box<dyn snapfaas::fs::BackingStore>);
```

Comment:
Can we put this in, say, src/fs/wrapper.rs?

Reply:
You are right.

Reply:
AsRef is a trait, so `impl<T: AsRef<B>, B: BackingStore> BackingStore for T`. The problem is that `B` isn't used anywhere, making it an unconstrained type parameter. Is there a workaround for this?